### PR TITLE
BEMHTML: Support escaped mode #1406

### PR DESCRIPTION
**Pull request by tadatuta**

### CODE REVIEW

File: .enb/tasks/tmpl-specs.js [hunk 1]
```diff
@@ -65,6 +79,16 @@ function configure(helper, platform) {
                     },
                     engineOptions : { elemJsInstances : true }
                 }
+            },
+            bemhtmlEscaped : {
+                tech : 'enb-bemxjst/techs/bemhtml',
+                options : {
+                    sourceSuffixes : ['bemhtml', 'bemhtml.js'],
+                    engineOptions : { escapeContent : true },
```
Comment (qfox) [inline .enb/tasks/tmpl-specs.js]: А почему без `elemJsInstances : true`? Или зачем они выше?

File: package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "enb-bem-specs": "^0.9.0",
     "enb-bem-techs": "^2.1.0",
     "enb-bem-tmpl-specs": "^1.1.0",
-    "enb-bemxjst": "^6.3.0",
+    "enb-bemxjst": "^6.5.3",
     "enb-bemxjst-i18n": "1.0.0-beta3",
```
Comment (qfox) [inline package.json:35]: It uses enb-bemxjst@2.1.0
Why we still using it?

File: .enb/tasks/tmpl-specs.js (hunk 1 above)
Comment (tadatuta) [reply .enb/tasks/tmpl-specs.js]: добавил, спасибо

File: package.json (hunk 2 above)
Comment (tadatuta) [reply package.json:35]: discussed in chat (package should be updated but we don't have time for it now)

**Pull request merged**